app/src/androidTest/java/com/github/sdp_begreen/begreen/firebase/meetingServices/MeetingPhotoServiceTest.kt

```diff
@@ -56,7 +58,7 @@ class MeetingPhotoServiceTest {
             runTest {
                 MeetingPhotoServiceImpl.addMeetingsPhoto(
                     " ",
-                    PhotoMetadata(),
+                    TrashPhotoMetadata(takenBy = "not null"),
```

Comment:
Is there a reason why we set the `takenBy` to "not null", whereas it was left as null before ? I am not sure to understand